### DISCUSSION TITLE
Welcome to igel Discussions!

### DISCUSSION THREAD

**Discussion by nidhaloff** [nidhaloff/igel#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Andrewwheeler89** [comment]
Is there any way to upgrade thousands of IGELs to 11.7.100.01? Any help or advice would be greatly appreciated.
